conf/galaxy_local.config

```diff
@@ -0,0 +1,20 @@
+docker {
+  enabled = true
+}
+
+params {
+  max_memory = 120.GB
+  max_cpus = 20
+  container='tristankast/cage-seq:latest'
+}
+
+process {
+  $star{
+  cpus = 20
+  memory = 120.GB
+  }
+  $makeSTARindex{
+  cpus = 20
+  memory = 120.GB
+  }
+}
```

Comment:
What is this config doing? Can we delete it too?

Reply:
This is still an artifact from Tristan's version from the beginning of this workflow. Will remove it.